includes/API/Pages/Read/Response.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+ *
+ * This source code is licensed under the license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @package FacebookCommerce
+ */
+
+namespace SkyVerge\WooCommerce\Facebook\API\Pages\Read;
+
+defined( 'ABSPATH' ) or exit;
+
+use SkyVerge\WooCommerce\Facebook\API;
+
+/**
+ * Page API response object.
+ *
+ * @since 2.0.0-dev.1
+ */
+class Response extends API\Response  {
+
+
+	/**
+	 * Gets the page name.
+	 *
+	 * @since 2.0.0-dev.1
+	 *
+	 * @return string|null
+	 */
+	public function get_name() {
+
+		return $this->name;
```

Comment:
Will the API handle the exception properly if the property does not exist?

Reply:
It would return `null` if the property doesn't exist. 

I recently added a [story](https://app.clubhouse.io/skyverge/story/55072/handle-graph-api-errors) to handle API exceptions. I believe that if we don't get an API error, we should receive all the field that requested. If that's the case, if `perform_request()` returns a response object, it should have the fields we need.